mod_app/admin/note_admin.py

```diff
@@ -43,19 +43,49 @@ def safe_content(self, obj):
     safe_content.short_description = "Content"
 
 
-class VWIInline(s3BrowserButtonMixin, admin.TabularInline):
-    model = VisualWrittenInfluences.films.through
+class VisInline(s3BrowserButtonMixin, admin.TabularInline):
+    model = VisualInfluences.films.through
     extra = 1
     classes = [
         "grp-collapse",
         "grp-open",
     ]
-    verbose_name = "Visual & Written Influences"
-    verbose_name_plural = "Visual & Written Influences"
+    verbose_name = "Visual Influences"
```

Comment:
```suggestion
    verbose_name = "Visual Influence"
```

On the model you've defined verbose name as "Visual Influence" rather than "Visual Influences".

Reply:
yep, deliberate, since they're separate now I thought they might end up making new entries for each

Reply:
worst case I'm wrong and we change it back